exercises/I-array-properties/exercise.js

```diff
@@ -6,7 +6,7 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
+  return ( arr === names); // complete this statement
```

Comment:
This doesn't make sense. You're comparing the array passed as a parameter to a global names array. 

You should just be checking the array passed in to the function's length:

i.e return array.length === 0;